test-pad/test-pad

```diff
@@ -0,0 +1,313 @@
+#!/bin/sh
+
+. "../bin/libstd-ecm.sh"
+
+usage() {
+    echo "usage: $0 [-r repo] [-c cluster] [-v version | -b location ] [sbdh]
```

Comment:
Can we drop the extra info after the flag? I'd normally to keep it if we had the distinction between short and long help but we don't and we have a description of the flags below. This also keeps with consistency of the other scripts in the repo.

Reply:
The reason I have info around these is because these are the "Required" flags for the command to work, while the others or "Optional". It doesn't have anything to do with short vs long.

Reply:
I'm not suggesting it does but was siting that as an example of their usage historically. I'd prefer consistency over deviation. The other scripts check for required flags and fail not present. This should do the same.